src/modules/fancyzones/editor/FancyZonesEditor/Models/CanvasLayoutModel.cs

```diff
@@ -120,10 +120,16 @@ protected override void PersistData()
         {
             AddCustomLayout(this);
 
+            var canvasRect = CanvasRect;
```

Comment:
I guess this happens for blank, and that is why we use `WorkArea` ?

Reply:
This happens for any template canvas. We cannot set its size in the constructor as we do for custom layouts since it should fit any work area.